.circleci/config.yml

```diff
@@ -204,6 +217,9 @@ jobs: # a collection of steps
           test: "test/relayclient/RelaySelectionManager.test.ts"
       - store_artifacts:
           path: ~/gls/logs
+      - store_artifacts:
+          path: ~/rsksmart/rskj/rskj-core/build/libs/logs
+          destination: ~/rskj/logs
```

Comment:
may i ask why this was added only to test thread #6? 

(not entirely sure what the purpose of this is, but mainly curious)